README.md

```
@@ -1 +1,167 @@
+
+- ## Running-Record 생성 API 처리량 개선
+- ## AWS 환경에 배포 후 Spring Cloud Gateway 가 Eureka Client 를 찾지 못하는 문제
+- ## 사용자 / 인증인가 서버분리와 게이트웨이 순환참조 오류
+
+<aside>
+
+
+# ♻️ CONTRIBUTORS
+| 팀원명 | 포지션 | 담당 | 깃허브 링크 |
+| --- | --- | --- | --- |
+| 강민 | 팀원 | ▶ 러닝기록<br>- 러닝기록 서비스는 유저 ID, 달린 거리, 달린 시간, 평균 페이스를 입력받아 러닝 기록으로 저장하는 서비스입니다.<br>- 웨어러블 디바이스에 따라 러닝 기록의 형식이 약간씩 차이가 있을 수는 있지만, 우선 공통적으로 사용되는 기록을 기반으로 러닝 기록 형식을 구현하였습니다.<br>- 러닝 기록 생성, 단건 조회, 전체 목록 조회, 검색, 삭제를 구현하였습니다.<br>- 러닝 기록 생성시, 업적 확인을 위해 업적 서비스에 Kafka를 이용하여 러닝 기록 생성 이벤트를 보냅니다.<br>▶ 업적<br>- 업적 서비스는 러닝 기록 기반으로 업적을 확인해주는 서비스입니다.<br>- Kafka를 이용하여 러닝 기록 생성 이벤트를 받아 이벤트 내용인 러닝 기록 수치와 업적 기준 및 수치를 비교하여 업적 달성 여부를 판단합니다.<br>- 업적 달성시, 해당 이용자에게 알람을 보내기위해 kafka를 이용하여 알람 이벤트를 알람 서비스에 보냅니다.<br>- 업적 확인 외 업적 등록, 단건 조회, 전체 목록 조회, 사용자 별 조회, 검색, 삭제를 구현하였습니다. | <a href="https://github.com/singingsandhill"><img height="130px" width="130px" src="https://avatars.githubusercontent.com/u/14273642?v=4"/></a> |
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: br

(MD033, no-inline-html)